src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -61,18 +89,16 @@ export default class AgreementType extends Vue {
   @State(state => state.stateModel.incorporationAgreementStep.agreementType)
   readonly agreementTypeState: string | null
 
-  @Prop({ default: false })
-  private showErrorSummary: boolean
-
-  @Prop({ default: false })
-  private isSummary: boolean
+  @State(state => state.stateModel.originalIA)
+  readonly originalIA!: IncorporationFilingIF
```

Comment:
I think we should stop using State (since that it "looking into the store structure") and instead use getters.
I think Cam created a getter for Original IA already.
Please also use a getter for Agreement Type.

Reply:
Sure. Forgot about that. Will do that

Reply:
I don't think i implemented a Getter for the entirety of the Original IA, i actually did as Lekshmi did above. We could refactor that tho! See also line 180 in `YourCompany` if you do create an Original IA getter.

Reply:
Done

Reply:
Hah, she implemented it! Well done :)